gsoc/proposal-EfficientDataStorage.md

```diff
@@ -0,0 +1,112 @@
+## Google Summer of Code - Application Template
+
+### Student
+
+**Name:** Chaitanya Prakash Bapat, Chaitanya
+
+**Affiliation:** Sardar Patel Institute of Technology (University of Mumbai), Computer Engineering, Bachelor of Engineering (BE)
+
+**Location:** Mumbai, India and India Time Zone UTC+05:30
+
+**GitHub handle:** [@ChaiBapchya](https://github.com/ChaiBapchya)
+
+**Slack handle:** @chaitanyabapat
+
+**Email:** chai.bapat@gmail.com
+
+**Other Contact:**  
+
+**Twitter** : [@ChaiBapchya](https://twitter.com/ChaiBapchya)
+
+**LinkedIn** : [/in/chaitanyaprakashbapat](https://www.linkedin.com/in/chaitanyaprakashbapat)
+
+### Project Details
+
+**Project Title:**  
+
+Efficient Data Storage using Geo-temporal Clustering and Diff-based Storage
+
+**Project Abstract/Summary:**  
+
+Clustering the web pages (data) based on Geographical location and time in order to facilitate Machine Learning analysis.
+
+**Describe the need your project fills:**  
+
+_Broader Goal_ :- Better data management and a step towards creating Environment Data Warehouse
+
+The project will fulfill the following needs - 
+
+_Speed, Accuracy, Objectivity(Contextualize), Performance, Scalability_
+
+**Describe how your project meets this need:**  
+1. **Speed** - Due to clustering of data, query execution time reduces dramatically, helping increase processing speed.
+2. **Accuracy** - By partitioning the data into seperate containers, possiblity of accessing anomalous, incorrect data will be greatly reduced.
+3. **Objectivity** - Performing division based on Time and location brings objectivity into the picture. By objectivity I mean, the change belongs to which region and during which time span. By quantifying these 2 major components, analysis is enriched and more contextualized.
```

Comment:
I'm not sure I fully understand this usage of "objectivity." Could you expand on this or try rephrasing?